package.json

```diff
@@ -28,7 +28,7 @@
     "pre-commit": "lint-staged"
   },
   "dependencies": {
-    "@openstfoundation/mosaic.js": "0.10.0-alpha.4"
+    "@openstfoundation/mosaic.js": "https://github.com/OpenSTFoundation/mosaic.js.git#npm_integration_fixes"
```

Comment:
This should probably not be part of `origin` like this.

Reply:
Agree! 
Do we already have updated mosaic.js NPM with latest fixes?

Reply:
Not yet ☝️

Reply:
ok. Let me know when available. Will update 👍